evap/results/exporters.py

```diff
@@ -160,18 +160,22 @@ def filter_evaluations(semesters, evaluation_states, degrees, course_types, cont
         return evaluations_with_results, used_questionnaires, course_results_exist
 
     def write_headings_and_evaluation_info(
-        self, evaluations_with_results, semesters, contributor, degrees, course_types
+        self, evaluations_with_results, semesters, contributor, degrees, course_types, verbose_heading
     ):
         export_name = "Evaluation"
```

Comment:
```suggestion
        export_name = _("Evaluation")
```

Since you're at it: This could be translated.